backend/backoffice/admin.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+
+from backoffice.models import TalkEntry
+
+
+@admin.register(TalkEntry)
+class TalkEntryAdmin(admin.ModelAdmin):

```

Comment:
I also suggest to add the `list_filter` field to `TalkEntryAdmin`, so that we can filter by `display`.

```python
    list_filter = ("display",)
```

Reply:
This also seems to me to be the main use case in the first instance. It's done